### DISCUSSION TITLE
I am a Software Engineer. How I can join Session team?

### DISCUSSION THREAD

**Discussion by dzianisv** [oxen-io/session-desktop#2393] [Q&A] [answered]
Hi, I am a Software Engineer. I liked the session and I don't see alternatives for secure messaging. Are you hiring, do you need engineers? How I can join and help to contribute to this awesome product?

**Comment by KeeJef** [comment, marked as answer]
Please send your resume to kee at oxen dot io

**Reply by dzianisv** [reply]
Done
